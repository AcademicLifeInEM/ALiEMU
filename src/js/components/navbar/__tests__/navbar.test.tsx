import React from 'react';
import renderer from 'react-test-renderer';

import { Navbar, NavGroup, NavTab } from '../index';

const noOp = () => void 0;

it('should render correctly', () => {
    const tree = renderer.create(
        <Navbar>
            <NavGroup>
                <NavTab active onClick={noOp}>
                    One
                </NavTab>
                <NavTab onClick={noOp}>Two</NavTab>
            </NavGroup>
            <NavGroup align="right">
                <NavTab onClick={noOp}>Three</NavTab>
            </NavGroup>
        </Navbar>,
    );
    expect(tree).toMatchSnapshot();
});
