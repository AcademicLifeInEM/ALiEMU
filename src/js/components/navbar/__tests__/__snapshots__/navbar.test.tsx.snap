// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<nav
  className="navbar"
>
  <div
    className="container"
  >
    <div
      className="navGroup"
    >
      <button
        className="btn active"
        onClick={[Function]}
      >
        One
      </button>
      <button
        className="btn"
        onClick={[Function]}
      >
        Two
      </button>
    </div>
    <div
      className="navGroup alignRight"
    >
      <button
        className="btn"
        onClick={[Function]}
      >
        Three
      </button>
    </div>
  </div>
</nav>
`;
